Documentation/user-manual.txt

```diff
@@ -711,12 +711,12 @@ be used).
 *Location*:
 [icon="images/icons/warning2.png"]
 [WARNING]
-Dive locations are managed as a *separate* part of the dive log.
+Dive locations are managed as a *separate* part of the dive log (See the section entitled xref:S_DiveSiteManagement[Manipulating Dive Site Information]).
 The dive information in the *Notes* and *Equipment* tabs can therefore NOT be edited at the same
 time as the dive site information. Save all the other dive information (e.g.
 divemaster, buddy, protective gear, notes about the dive) by selecting _Apply changes_ on the
 *Notes* tab BEFORE editing the dive site information. Then supply a dive site name in the
-textbox labeled _Location_ on the *Notes* tab.
+textbox labeled _Location_ on the *Notes* tab. 
```

Comment:
There is a slight inconsistency between `labeled` and `labelled`. I believe but are OK (US vs. commonwealth), but perhaps unify in the same chapter.